chai.js

```diff
@@ -3496,16 +3496,27 @@ module.exports = function (chai, _) {
     if (msg) flag(this, 'message', msg);
     var expected = flag(this, 'object')
       , flagMsg = flag(this, 'message')
-      , ssfi = flag(this, 'ssfi');
+      , ssfi = flag(this, 'ssfi')
+      , contains = flag(this, 'contains');
     new Assertion(list, flagMsg, ssfi, true).to.be.an('array');
 
-    this.assert(
+    if(contains) {
```

Comment:
Sorry about this nitpicking but can you add a space between the `if` and the `(contains) {`?
`if (contains) {`
Just to keep style consistency.